tap_linkedin_ads/__init__.py

```diff
@@ -7,7 +7,7 @@
 from singer import metadata, utils
 from tap_linkedin_ads.client import LinkedinClient
 from tap_linkedin_ads.discover import discover, check_accounts_list
-from tap_linkedin_ads.sync import sync
+from tap_linkedin_ads.sync import sync as _sync
```

Comment:
There is a sync function also and the module also, which is conflicting hence changed the function import as _sync.